src/Dropdown.vue

```diff
@@ -15,7 +15,7 @@
     <slot name="before"></slot>
     <slot name="button">
       <button type="button" class="btn dropdown-toggle" :class="[btnType, btnWithBefore]" @keyup.esc="hideDropdownMenu()" :disabled="disabled">
-        {{ text }}
+        <span v-html="text"></span>
```

Comment:
Seems like this is from another PR?

Reply:
I rolled back these files to what's in master branch. Thanks.

Reply:
Ok please rebase the branch to the latest master branch, so that these changes will not appear in diffs.